gen/generate.go

```diff
@@ -326,13 +340,18 @@ func generateGenesisFiles(outDir string, protoVersion protocol.ConsensusVersion,
 		fmt.Fprintln(verboseOut, protoVersion, protoParams.MinBalance)
 	}
 
+	if rewardsBalance < protoParams.MinBalance {
```

Comment:
@barnjamin I feel like I'm not fully aware of the consequences, so I'm asking - Is the following behavior intentional?

It appears the highlighted check happens _after_ allocating staking amounts.  Implies the staked balance _may not_ match account balance.

Reply:
Resolving - I _think_ the concern is safe to resolve because fee sink + rewards pool are _not_ included in the total money calculation in `setupGenerateGenesisFiles`.